releasenotes/notes/runtime-stack-frame-507353b6b01490ad.yaml

```
@@ -0,0 +1,4 @@
+features:
+  - |
+    Crashtracker: This introduces a fallback to capture runtime stack frames when
+    Python's `_Py_DumpTracebackThreads` function is not available.
```

```suggestion
    Python's ``_Py_DumpTracebackThreads`` function is not available.
```

reno expects rST not markdown